README.md

```diff
@@ -77,3 +77,11 @@ Your feedback is our motivation to move on. Please let us know your questions, c
 ## License
 
 [Apache License 2.0](https://github.com/sql-machine-learning/sqlflow/blob/develop/LICENSE)
+
+## Published
```

Comment:
Published or Publication?

Reply:
Not sure if Youtube videos are publications, so I used the word published.